docs/config/index.md

```diff
@@ -227,6 +227,7 @@ Custom reporters for output. Reporters can be [a Reporter instance](https://gith
   - `'dot'` -  show each task as a single dot
   - `'junit'` - JUnit XML reporter
   - `'json'` -  give a simple JSON summary
+  - path of a custom reporter (e.g. `'/path/to/reporter.ts'`, `'@company/reporter'`)
```

Comment:
@sheremet-va do you think we should list the examples followed by the description kind of like the ones above this? 

`'/path/to/reporter.ts'` or `'@company/reporter'` - path of a custom reporter